### ISSUE TITLE
Create an update routine which fetches new map data and converts them to mbtiles

### ISSUE THREAD

**Issue by dephiloper** [LB31/HikingForNerds#50]
As explained in this [issue](https://github.com/LB31/HikingForNerds/issues/27) by @NovalisTheMagician we are able to fetch map data from geofabrik and convert it to mbtiles. Therefore a script will be provided by @NovalisTheMagician which has to be integrated into the server workflow.

**Comment by dephiloper** [comment]
depends on #51

**Comment by dephiloper** [comment]
Closed because it is no longer relevant.
